prac_06/guitars.py

```diff
@@ -0,0 +1,19 @@
+from prac_06.guitar import Guitar
+
+print("My Guitars!")
+guitars = []
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = int(input("Cost: "))
+    guitars.append(Guitar(name, year, cost))
+    print(f"{name} ({year}) : ${cost} added.")
+    name = input("Name: ")
+
+i = 1
+for guitar in guitars:
```

Comment:
You can use the enumerate function to create a counter in your loops. It will also iterate through them automatically. e.g.
for i, guitar in enumerate(guitars)